perfmetrics/scripts/read_cache/setup.sh

```diff
@@ -64,7 +64,7 @@ sed -i 's/define \+FIO_IO_U_PLAT_GROUP_NR \+\([0-9]\+\)/define FIO_IO_U_PLAT_GRO
 cd -
 
 # Install and validate go.
-version=1.21.3
+version=1.22.0
```

Comment:
Let's add this also to the internal doc

Reply:
@ashmeenkaur  Made the change you requested separately (outside the scope of this PR) as internal documentation is not in this repo.